src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -97,7 +97,8 @@ protected StudentAttributes getStudent(StudentAttributes student) {
     }
 
     protected void removeAndRestoreTypicalDataBundle() {
-        removeAndRestoreDataBundle(typicalBundle);
+        DataBundle dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
```

Comment:
Find out why, and fix the actual cause. `removeAndRestoreDataBundle(typicalBundle);` should have no difference from `DataBundle dataBundle = getTypicalDataBundle(); removeAndRestoreDataBundle(dataBundle);`.